src/main/java/ru/hogwarts/school/service/FacultyService.java

```diff
@@ -0,0 +1,53 @@
+package ru.hogwarts.school.service;
+
+import org.springframework.stereotype.Service;
+import ru.hogwarts.school.model.Faculty;
+import ru.hogwarts.school.repository.FacultyRepository;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class FacultyService {
+    private final FacultyRepository repository;
+
+    public FacultyService(FacultyRepository repository) {
+        this.repository = repository;
+    }
+
+    public Faculty createFaculty(Faculty faculty) {
+        return repository.save(faculty);
+    }
+
+    public List<Faculty> createFaculties(List<Faculty> faculties) {
+        return repository.saveAll(faculties);
+    }
+
+    public Faculty getFaculty(Long id) {
+        if (repository.findById(id).isPresent()) {
+            return repository.findById(id).get();
+        } else {
+            throw new NoSuchSomeObjectException(" - " + id + " does not exist");
+        }
```

Comment:
repository.findById(id).orElseThrow(() -> new NoSuchSomeObjectException(" - " + id + " does not exist"))